node/src/components/storage/lmdb_ext.rs

```diff
@@ -157,15 +164,44 @@ where
     }
 }
 
+/// Serializes `value` into the buffer.
+/// In case the `value` is of the `UnbondingPurse` type it uses the specialized
+/// function to provide compatibility with the legacy version of the `UnbondingPurse` struct.
+/// See [`serialize_unbonding_purse`] for more details.
+pub(crate) fn serialize_internal<V: 'static + Serialize>(
+    value: &V,
+) -> Result<Vec<u8>, LmdbExtError> {
+    let buffer = if TypeId::of::<UnbondingPurse>() == TypeId::of::<V>() {
+        serialize_unbonding_purse(value)?
+    } else {
+        serialize(value)?
+    };
+    Ok(buffer)
+}
+
+/// Deerializes an object from the raw bytes.
```

Comment:
```suggestion
/// Deserializes an object from the raw bytes.
```

Reply:
Resolved in 8100ff7148b1f8eb4dd55aef8f878be2995bbdc9